src/extensions/default/AutoUpdate/main.js

```diff
@@ -104,9 +129,12 @@ define(function (require, exports, module) {
      * @param {object}   msgObj - json containing - {
      *                          fn - function to execute on Brackets side
      *                          args - arguments to the above function
+     *                          requester - ID of the current requester domain
      */
     function receiveMessageFromNode(event, msgObj) {
-        functionMap[msgObj.fn].apply(null, msgObj.args);
+        if (domainID === msgObj.requester) {
```

Comment:
seems like this will always be true for a particular session?

Reply:
This will always be true for a particular window session, but different for different windows.